plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -60,10 +61,19 @@ export default {
               return dasherize(slugifyChannel(this.channel));
             }
 
+            @computed("chatService.activeChannel")
             get classNames() {
-              return this.channel.current_user_membership.muted
-                ? "sidebar-section-link--muted"
-                : "";
+              const classes = [];
+
+              if (this.channel.current_user_membership.muted) {
+                classes.push("sidebar-section-link--muted");
+              }
+
+              if (this.channel?.id === this.chatService.activeChannel?.id) {
```

Comment:
Nitpick, but we should either check for `this.channel?` in both places here, or in neither. I'd go with neither.

Reply:
yes right we should always have the channel here, will change thx